src/client/AM.PA.MonitoringTool/ActivityMapper/ProcessToActivityMapper.cs

```diff
@@ -71,15 +70,19 @@ private static ActivityCategory GetCategory(string processName, string windowNam
             try
             {
                 if (string.IsNullOrEmpty(processName) && string.IsNullOrEmpty(windowName))
+                {
                     return ActivityCategory.Unknown;
+                }
 
-                if (windowName != null) windowName = windowName.ToLower(CultureInfo.InvariantCulture);
-                if (processName != null) processName = processName.ToLower(CultureInfo.InvariantCulture);
+                windowName = windowName?.ToLower();
+                processName = processName?.ToLower();
```

Comment:
Why is the parameter `CultureInfo.InvariantCulture` no longer required?

Reply:
The project I created is .NET Standard (as opposed to .NET Framework) and the version I used imho does not even know about culture or at least does not provide the ToLower functions using Cultures. This is why I had to change the code here.
Cultures most certainly only matter in interpretable data like DateTime, float, double or decimal. For windowName and ProcessName it wouldn't. The method without overload would default to InvariantCulture anyway.

Reply:
Good to know, didn't know that. I've added them everywhere after the Microsoft Static Code Analysis tool suggested that 😁